edk2toolext/base_abstract_invocable.py

```diff
@@ -97,6 +101,7 @@ def Invoke(self):
         ''' Main process function.  Should not need to be overwritten '''
 
         self.ParseCommandLineOptions()
+        self.InputParametersConfigured()
```

Comment:
Should we setup logging before we call this function? Maybe we should be more clear about what functions have logging setup. Right now it's only ParseCommandLineOptions

Reply:
agree.  Moved to after logging